service/lib/ops/policyOps.js

```diff
@@ -39,16 +39,26 @@ function insertPolicies (client, policies, cb) {
   client.query(sql, cb)
 }
 
-function deletePolicies (client, ids, cb) {
-  client.query(SQL`DELETE FROM policies WHERE id = ANY (${ids})`, utils.boomErrorWrapper(cb))
+function deletePolicies (client, ids, orgId, cb) {
+  client.query(SQL`DELETE FROM policies WHERE id = ANY (${ids}) AND org_id=${orgId}`, utils.boomErrorWrapper(cb))
```

Comment:
But if we return an error code, it means the `DELETE` should not proceed, correct ?
Also, what is our convention, 400 or 404 ?

From a user's perspective, I would like a well behaved API to try to execute my query and then return to me which ones of the IDs failed to delete if any. Or simpler than that, a count of how many policies were deleted.

Also, since orgs are completely secluded, how likely is it that for this scenario to really happen ?